### DISCUSSION TITLE
Deinstallation of node-red in home assistant

### DISCUSSION THREAD

**Discussion by Prickelkoenig** [hassio-addons/addon-node-red#1563] [General] [answered]
I dont need no longer node-red. Now i want to delete it.
I  delete it on HACS and after that i restart home assistant.
But node-red is not gone.
What can i do?

**Comment by frenck** [comment, marked as answer]
Uninstall the add-on... And done.

**Reply by Prickelkoenig** [reply]
I have uninstall this in HACs as i wrote, but it is still in the sidebar and launches when i click on it. (Sorry i am a beginner in Home Assistant).
I am running node-red on a raspi4.

**Reply by frenck** [reply]
This is not HACS, HACS doens't provide add-ons.

You are mixing up add-ons & integrations. What you removed from HACS is probably the custom integration for Node-RED. What you see in the sidebar, is Node-RED itself, which is installed in HA as an add-on. Setings -> addons.

../Frenck

**Comment by Prickelkoenig** [comment]
In HACS it is gone, therefore i assumed, that it is unstalled, but its still in the sidebar.

**Comment by Prickelkoenig** [comment]
Thank you. I have deinstall it in Settings and its gone. 
Problem solved. :-)
